tests/fixtures/markdown/export/expected.md

```diff
@@ -0,0 +1,206 @@
+</tr>
+<tr>
+<td> processed_timestamp</td>
+<td>timestamp</td>
+<td><i>The timestamp when the record was processed by the data platform.</i>
+<li><code>required</code></li>
+<li><b>config:</b> {'jsonType': 'string', 'jsonFormat': 'date-time'}</li></td>
+</tr>
+</table>
+
+<table>
+<tr>
+<th colspan=3>LINE_ITEMS<br/>
+<i>A single article that is part of an order.</i></th>
+</tr>
+<tr>
+<td> line_item_id</td>
+<td>text</td>
+<td><i>Primary key of the lines_item_id table</i>
+<li><code>required</code></li></td>
+</tr>
+<tr>
+<td> order_id</td>
+<td>None</td>
+<td><i>No description.</i>
+<li><b>ref:</b> #/definitions/order_id</li>
+<li><b>references:</b> orders.order_id</li></td>
+</tr>
+<tr>
+<td> sku</td>
+<td>None</td>
+<td><i>The purchased article number</i>
+<li><b>ref:</b> #/definitions/sku</li></td>
+</tr>
+</table>
+
+
+## Definitions
+<table>
+<tr>
+<th>Name</th>
+<th>Type</th>
+<th>Domain</th>
+<th>Attributes</th>
+</tr>
+<tr>
+<td>order_id</td>
+<td>text</td>
+<td></td>
+<td>
+<i>An internal ID that identifies an order in the online shop.</i>
+<li><b>title:</b> Order ID</li>
+<li><b>format:</b> uuid</li>
+<li><code>pii</code></li>
+<li><b>classification:</b> restricted</li>
+<li><b>tags:</b> ['orders']</li>
+<li><b>examples:</b> ['243c25e5-a081-43a9-aeab-6d5d5b6cb5e2']</li>
+</td>
+</tr>
+<tr>
+<td>sku</td>
+<td>text</td>
+<td></td>
+<td>
+<i>A Stock Keeping Unit (SKU) is an internal unique identifier for an article. 
+It is typically associated with an article's barcode, such as the EAN/GTIN.
+</i>
+<li><b>title:</b> Stock Keeping Unit</li>
+<li><b>pattern:</b> ^[A-Za-z0-9]{8,14}$</li>
+<li><b>tags:</b> ['inventory']</li>
+<li><b>links:</b> {'wikipedia': 'https://en.wikipedia.org/wiki/Stock_keeping_unit'}</li>
+<li><b>examples:</b> ['96385074']</li>
+</td>
+</tr>
+</table>
+
+## Service levels
+
+### Availability
+<i>The server is available during support hours</i>
+<li><b>percentage:</b> 99.9%</li>
+
+### Retention
+<i>Data is retained for one year</i>
+<li><b>period:</b> P1Y</li>
+
+### Latency
+<i>Data is available within 25 hours after the order was placed</i>
+<li><b>threshold:</b> 25h</li>
+<li><b>sourceTimestampField:</b> orders.order_timestamp</li>
+<li><b>processedTimestampField:</b> orders.processed_timestamp</li>
+
+### Freshness
+<i>The age of the youngest row in a table.</i>
+<li><b>threshold:</b> 25h</li>
+<li><b>timestampField:</b> orders.order_timestamp</li>
+
+### Frequency
+<i>Data is delivered once a day</i>
+<li><b>type:</b> batch</li>
+<li><b>interval:</b> daily</li>
+<li><b>cron:</b> 0 0 * * *</li>
+
+### Support
+<i>The data is available during typical business hours at headquarters</i>
+<li><b>time:</b> 9am to 5pm in EST on business days</li>
+<li><b>responseTime:</b> 1h</li>
+
+### Backup
+<i>Data is backed up once a week, every Sunday at 0:00 UTC.</i>
+<li><b>interval:</b> weekly</li>
+<li><b>cron:</b> 0 0 * * 0</li>
+<li><b>recoveryTime:</b> 24 hours</li>
+<li><b>recoveryPoint:</b> 1 week</li>
```

Comment:
I would expect to habe the markdown markup and not html tags. Can you change that, please?

Reply:
I changed the way to construct list elements. 
I used `•` char instead of list `- ` because Markdown list aren't supported in table cell.